DP02_17_Galaxy_Photometry.ipynb

```diff
@@ -0,0 +1,1077 @@
+{
```

Comment:
<p>"while the Kron magnitude approaches the cModel magnitude for fainter galaxies" -- the x-axis is Kron radius (rather than a magnitude), and I wonder if there're some small galaxies that are bright?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<ol><li>whoops, this was meant to be smaller galaxies (but i guess they are all tending to be fainter as well). Fixed.</li><li>Thanks, added.</li><li>Good point, fixed.</li><li>Thanks for catching this mistake (I missed it). Fixed</li></ol><p> </p>